src/api/requests.tsx

```diff
@@ -0,0 +1,356 @@
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+//path boards/${boardId}/columns/${columnId}/tasks
+export const createTask = async (token: string, path: string, task: ITask) => {
+  return await request(`${base}/${path}`, 'POST', JSON.stringify(task), {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+export const getTaskById = async (
+  token: string,
+  boardId: string,
+  columnId: string,
+  taskId: string
+) => {
+  return await request(
+    `${base}/boards/${boardId}/columns/${columnId}/tasks/${taskId}`,
+    'GET',
+    null,
+    {
+      'Content-type': 'application/json',
+      Authorization: `Bearer ${token}`,
+    }
+  );
+};
+
+//path boards/${boardId}/columns/${columnId}/tasks/${taskId}
+export const updateTask = async (token: string, path: string, task: ITask) => {
+  return await request(`${base}/${path}`, 'PUT', JSON.stringify(task), {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+//path boards/${boardId}/columns/${columnId}/tasks/${taskId}
+export const deleteTask = async (token: string, path: string) => {
+  return await request(`${base}/${path}`, 'DELETE', null, {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+export const getTaskSet = async (token: string) => {
+  return await request(`${base}/taskSet`, 'GET', null, {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+export interface ITaskSet {
+  _id: string;
+  order: number;
+  columnId: string;
+}
+export const updateTaskSet = async (token: string, taskSet: ITaskSet[]) => {
+  return await request(`${base}/tasksSet`, 'PATCH', JSON.stringify(taskSet), {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+export const getBoardTaskSet = async (token: string, boardId: string) => {
+  return await request(`${base}/tasksSet/${boardId}`, 'GET', null, {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+//Points
+interface IPoint {
+  title: string;
+  taskId: string;
+  boardId: string;
+  done: boolean;
+}
+
+export const getPoints = async (token: string) => {
+  return await request(`${base}/points`, 'GET', null, {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+export const createPoints = async (token: string, point: IPoint) => {
+  return await request(`${base}/points`, 'POST', JSON.stringify(point), {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+export const getPointsTaskId = async (token: string, taskId: string) => {
+  return await request(`${base}/points/${taskId}`, 'GET', null, {
+    'Content-type': 'application/json',
+    Authorization: `Bearer ${token}`,
+  });
+};
+
+export const updatePoint = async (token: string, pointId: string, title: string, done: boolean) => {
```

Comment:
try to avoid accepting more than 3 arguments in your functions
if you really have to pass all these values, then prefer objects instead

something like:

```ts
type Payload = {
  pointId: string;
  title: string;
  done: boolean;
}

export function updatePoint(token: string, payload: Payload) { }
export function deletePoint(token: string, payload: Pick<Payload, 'pointId'>) { }
```